Octokit.Reactive/Clients/ObservableRepositoryCommentsClient.cs

```diff
@@ -140,19 +229,46 @@ public IObservable<CommitComment> Update(string owner, string name, int number,
         }
 
         /// <summary>
+        /// Updates a specified Commit Comment.
+        /// </summary>
+        /// <remarks>http://developer.github.com/v3/repos/comments/#update-a-commit-comment</remarks>
+        /// <param name="repositoryId">The ID of the repository</param>
+        /// <param name="number">The comment number</param>
+        /// <param name="commentUpdate">The modified comment</param>
+        /// <returns>A <see cref="IObservable{CommitComment}"/> of <see cref="CommitComment"/> for the specified commit comment.</returns>
+        public IObservable<CommitComment> Update(int repositoryId, int number, string commentUpdate)
+        {
+            Ensure.ArgumentNotNull(commentUpdate, "commentUpdate");
+
+            return _client.Update(repositoryId, number, commentUpdate).ToObservable();
+        }
+
+        /// <summary>
         /// Deletes the specified Commit Comment
         /// </summary>
         /// <remarks>http://developer.github.com/v3/repos/comments/#delete-a-commit-comment</remarks>
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="number">The comment id</param>
-        /// <returns></returns>
+        /// <returns>A <see cref="IObservable{Unit}"/></returns>
```

Comment:
Add a comment on the end here "... indicating completion" because the `Unit` part really isn't interesting here...